jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -84,23 +73,18 @@ private SettlerImageMap() {
 	 * @param file
 	 * 		The file to read from.
 	 */
-	private void readFromFile(InputStream file) throws IOException {
-		int[][][][] priorities = new int[this.types][this.actions][this.materials][this.directions];
-
-		// add pseudo entry.
-		addEntryToMap(priorities, null, null, null, null, DEFAULT_ITEM, -1);
-
-		readFromFile(file, priorities);
-	}
-
-	private void readFromFile(InputStream file, int[][][][] priorities) throws IOException {
+	private void readFromStream(InputStream file) throws IOException {
```

Comment:
This method can be inlined. Both, the caller and this method, are now two-liners.